foundations/javascript_basics/fundamentals-4.md

```diff
@@ -1,3 +1,9 @@
+### Introduction
+
+One of the challenges of programming is dealing with large amounts of data. For example, if you want to store the names of all the students in your class, how would you do it? You could create a variable for each name, but that would be tedious and inefficient. It'll also be hard to manage and update. What if you want to change or access the names later?
+
+Luckily, there is a better way to handle this problem. In this lesson, you’ll learn about arrays, which are data structures that can store multiple values in a single variable. They are very useful for organizing and manipulating large amounts of data. You’ll also learn about loops, control structures that allow you to execute a block of code repeatedly. Loops are very handy for performing the same operation on each element of an array. Finally, you’ll be introduced to Test-Driven Development (TDD), the practice of writing tests for your code before you create it.
```

Comment:
```suggestion
Luckily, there is a better way to handle this problem. In this lesson, you’ll learn about arrays, which are data structures that can store multiple values in a single variable. Arrays are very useful for organizing and manipulating large amounts of data. You’ll also learn about loops, which are control structures that allow you to execute a block of code repeatedly. Loops are very handy for performing the same operation on each element of an array. Finally, you’ll be introduced to Test-Driven Development (TDD), which is the practice of writing tests for your code before you write the code itself.
```

Wonderful job Cake! Considering learners are very new, I reverted back to being a bit more descriptive and repetitive as well. I think being extra clear what is meant with these new terms is crucial.

Reply:
I don't have strong feelings about either, so I'll follow your feedback.